bootable-jar/runtime/src/main/java/org/wildfly/core/jar/runtime/_private/BootableJarLogger.java

```diff
@@ -162,4 +162,7 @@
 
     @Message(id = Message.NONE, value = "Display the content of the Galleon configuration used to build this bootable JAR")
     String argDisplayGalleonConfig();
+
+    @Message(id = Message.NONE, value = "Path to a CLI script to execute")
```

Comment:
maybe specify that this option is when running the bootable jar (to avoid being confused with CLI scripts that are executed when the jar is built).

Reply:
Good point!